webpack.config.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+module.exports = {
+  entry: "./src/ping-centre.js",
+  output: {
+    path: "./dist",
+    filename: "ping-centre.min.js",
+    // exports entry file to a global variable
+    libraryTarget: "var",
+    // name of the global variable
+    library: "Ping-Centre"
```

Comment:
Ah, should this be "PingCentre" here? Not sure if this is what was causing the `var Ping-Centre = ...;` business.